arbitrum-docs/bold/concepts/bold-technical-deep-dive.md

```diff
@@ -0,0 +1,231 @@
+
+Edges also have an **inherited timer**, which is the sum of its unrivaled timer + the minimum inherited timer of an edge's children (recursive definition). Once one of the top-level edges that initiated a challenge has achieved an inherited timer >= a CHALLENGE_PERIOD (6.4 days), it can be confirmed. At this point, its assertion can also be confirmed as its associated challenge has completed. A minor but important detail is that edges also inherit the time their claimed assertion was unrivaled.
+
+We believe timer inheritance from ancestor edges is fundamentally broken. Honest edges could have evil ancestors or vice versa, and edges could steal/claim timer credit from others to which they should not be entitled. The [research specification](https://arxiv.org/abs/2404.10491) goes in-depth into the proven lemmas of timer inheritance and why children's inheritance solves critical attack vectors.
+
+#### Cached timer updates
+
+An edge's "inherited timer" value exists on-chain and can be updated via a transaction. Given it is a recursive definition, it can be updated via multiple transactions. First, the lowermost edges have their timers updated, then their parents, etc., up to the top. Validators can track information locally to avoid sending wasteful transactions and only propagate updates once they are confident their edge is confirmable by time.
+
+#### Confirmation
+
+Once an edge has a total on-chain timer greater than or equal to a challenge period, it can be confirmed via a transaction. Not all edges need to be confirmed on-chain, as simply the top-level block challenge edge is enough to confirm the claimed assertion and resolve a dispute. A challenge is not complete at the one-step proof. It is only complete once the claimed assertion of a challenge is confirmed.
+
+### Bonding in challenges
+
+To create a challenge, there must be a fork in the Arbitrum assertion chain smart contract. A validator that wishes to initiate a challenge must then post an “edge” claiming a history of block hashes from the parent assertion to the claimed assertion they believe is correct. To do so, they need to put up some value called a "challenge bond". Note that to open a new assertion-level challenge, the challenge bond is equal to the assertion bond for <a data-quicklook-from="arbitrum-one">Arbitrum One</a>.
+
+<a data-quicklook-from="challenge">Challenge</a> bonds are named as such because they are bonds required
+for opening challenges. The mechanism of how challenge bond economics are decided is contained in the
+economics deep dive, which also explains the cost profile and spam prevention in BoLD. In short, the
+actual cost of a bond encompasses information such as off-chain costs + gas costs + grieving ratios between
```

Comment:
I anticipate that "grieving ratios" may be a new term for some readers. I would either briefly define it here or in the glossary at the top. Alternatively, you could just not enumerate the information a bond encompasses and leave it to the linked document to get into the details.